TestApps/SIL.Windows.Forms.TestApp/TestAppForm.cs

```diff
@@ -21,6 +22,8 @@
 using SIL.Windows.Forms.SettingProtection;
 using SIL.Windows.Forms.WritingSystems;
 using SIL.WritingSystems;
+using SIL.Media;
+using Microsoft.VisualBasic.Devices;
```

Comment:
I can't figure out what of Microsoft.VisualBasic.Devices is being used. I suppose if I had the code in an IDE I would comment it out and see what turns red :-)